InsiteTeamTask/Services/Attendance/AttendenceViewService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AutoMapper;
+using InsiteTeamTask.Logic.Attendence;
+using InsiteTeamTask.Models;
+
+namespace InsiteTeamTask.Services
+{
+    public class AttendenceViewService : IAttendenceViewService
+    {
+        private readonly IMapper _mapper;
+        private readonly IAttendenceService _attendenceService;
+
+        public AttendenceViewService(IMapper mapper, IAttendenceService attendenceService)
+        {
+            _mapper = mapper;
+            _attendenceService = attendenceService;
+        }
+        public AttendenceViewModel GetAttendence(int gameNumber, int seasonNumber)
+        {
+            var dto = _attendenceService.GetAttendenceForGameNumberAndSeasonNumber(gameNumber, seasonNumber);
+
+            return _mapper.Map<AttendenceViewModel>(dto);
```

Comment:
I've mentioned this elsewhere, but this service acts as nothing more than a middleman for casting between identical objects. You'd be better off injecting your `IAttendanceService` into your controller